Reg: Some maintenance of the tool

harish0201 opened the pull request:
Hey,

I came across this awesome work a few hours earlier.

Noted that you were using BWA to align which was taking a really long time on a genome of ~500Mbp.

I have modified it to use minimap2 instead of BWA, changed label size and location in circos conf. I've also changed the minbundlesize to 50kbp so as to remove smaller alignments and clean up the plot a bit more.

I've also removed circos 0.6xx version, so that you can use circos tools from bioconda for easier maintenance.

Currently, I'm trying out to figure a way so that the *agp.bed file doesn't get removed, but the easiest way is to 
`cp $@ $@.bak` in the code just after generating the bed.

I'd appreciate it if you could go through it and merge, as it saved me tons of headache today.

JustinChu commented:
Hey thanks for doing this. I've been meaning to replace bwa with minimap for a while.

I think adding instructions for installing circos from homebrew/linuxbrew and conda in the readme would be helpful.

I'm not sure if changing the default minbundles is needed, since you can easily change it. The values really depend on the size of the genomes after all. Then again, it isn't a big deal to change it either.

harish0201 commented:
Hey,

Thanks for the great help!

I changed minbundles on my end, because I was working with a highly contiguous genome, I would have also changed the circos config as well.

Regarding, installation of circos, I agree that making a disclaimer on conda would be really helpful!
